lib/concentrate/group_filter/scheduled_stop_times.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Concentrate.GroupFilter.ScheduledStopTimes do
+  @moduledoc """
+  Uses the static GTFS schedule to fill in missing arrival/departure times on stop time updates
+  that have specific `status` values.
+
+  The desired status values must be set in app configuration. If no status values are configured,
+  enabling this filter has no effect. Example:
```

Comment:
```suggestion
  enabling this filter has no effect. 
  
  Example:
```

Previous version reads like this is an example of when it has no effect.

Reply:
Hm, good point.